task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -35,13 +35,21 @@
 from airflow.sdk.api import client as sdk_client
 from airflow.sdk.api.datamodels._generated import TaskInstance
 from airflow.sdk.api.datamodels.activities import ExecuteTaskActivity
-from airflow.sdk.execution_time.comms import ConnectionResult, GetConnection, GetVariable, VariableResult
+from airflow.sdk.execution_time.comms import (
+    ConnectionResult,
+    GetConnection,
+    GetVariable,
+    PatchTIToDeferred,
+    VariableResult,
+)
 from airflow.sdk.execution_time.supervisor import WatchedSubprocess, supervise
 from airflow.utils import timezone as tz
 
 if TYPE_CHECKING:
     import kgb
 
+TI_ID = uuid7()
```

Comment:
To make it predictable. It will be needed to assert the task instance endpoints as we will assert the id too